web/libs/editor/src/tags/control/Taxonomy/Taxonomy.scss

```diff
@@ -81,4 +81,8 @@ body :global(.ant-select-multiple .ant-select-selection-item-remove) {
 
 body :global(.ant-select-arrow) {
   color: var(--color-neutral-icon);
+}
+
+body :global(.ant-select-single .ant-select-selection-item) {
+  color: var(--color-neutral-content);
```

Comment:
Why is a change to the Taxonomy tag required for this PR?

Reply:
I think so yes, without that the selection items look incorrect in  dark mode